python/ray/rllib/utils/sampler.py

```diff
@@ -252,37 +191,30 @@ def _env_runner(env, policy, num_local_steps, horizon, obs_filter):
                 action = np.concatenate(action, axis=0).flatten()
 
             # Collect the experience.
-            rollout.add(
+            batch_builder.add_values(
                 obs=last_observation,
                 actions=action,
                 rewards=reward,
                 dones=terminal,
-                features=last_features,
                 new_obs=observation,
```

Comment:
IDK where to leave this note, but we're actually doubling the number of states we need to send here (`observation` and `last_observation`). In a later optimization, we should consider addressing this (I can put this on TODOs)